### PR TITLE
Prepare testing for cert-manager v1.2 release

### PR DESCRIPTION
**Pull request by jakexks**

### CODE REVIEW

File: config/jobs/cert-manager/cert-manager-presubmits.yaml
Comment (wallrj) [inline config/jobs/cert-manager/cert-manager-presubmits.yaml:541]: Incremented all the release branch numbers. 
So our PRs will be tested after merging the branch into master and also into release-1.3

File: config/jobs/cert-manager/release-next/cert-manager-release-next-periodics.yaml
Comment (wallrj) [inline config/jobs/cert-manager/release-next/cert-manager-release-next-periodics.yaml:11]: Nightly tests to run against the release-1.3 branch.

File: config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml
Comment (wallrj) [inline config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml:11]: Nightly tests to now run against the 1.2 release.
 * https://testgrid.k8s.io/jetstack-cert-manager-previous

File: config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml
Comment (wallrj) [inline config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml:371]: And we removed all the now unsupported K8S versions from release-previous, because it now tests 1.2

**Review by wallrj — COMMENTED** [review]
This all looks good to me @jakexks 

But question for @munnerz and @meyskens....what about when we start backporting patches to 1.1? How will we run e2e tests against the release-1.1 branch which needs to still be tested with all the legacy versions of Kubernetes.

**Comment by munnerz** [comment]
We only backport back a single version, right? So the only version eligible for cherry picks as of today is 1.2 (so 1.1 testing can be retired).

**Comment by jakexks** [comment]
/hold
To be merged as soon as 1.1.1 is released

**Review by wallrj — APPROVED** [review]
/lgtm

**Pull request merged**
